src/style.css

```diff
@@ -34,3 +58,50 @@ p {
 .red-border {
   border: 2px solid red;
 }
+
+body {
```

Comment:
[OPTIONAL] You can introduce scrolling to the whole page using `overflow: scroll` to allow users to scroll down the page to view the rest of the table.

![image](https://user-images.githubusercontent.com/38088818/195711956-33c983b7-abbf-419c-b7eb-aa1946ee4fb4.png)